app/assets/scripts/components/bar-chart.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Bar } from 'react-chartjs-2';
 
-export default function BarChart({ title, frequency, xAxisLabels }) {
-  const data = {
+import { round } from '../utils/format';
+
+export default function BarChart({ data, xAxisLabels }) {
+  const series = {
     labels: xAxisLabels,
     datasets: [
       {
-        label: 'Measurements',
-        data: frequency,
+        label: 'average',
```

Comment:
What is this label used for? just wondering if it should be passed as props or ok to hardcode?

Reply:
it appears on the legend (currently hidden) and the tooltips. Passing as props is a good idea in this case, as it could just as well be the count (see [this line](https://github.com/openaq/openaq.org/pull/450/files/0267fba245319314df603234823dcaaf7753dd20#diff-afda5e67a8c150e5d2106efb699b7eb2f95fdc0c7d8fca99f321a724fd3005b6R248))